locale-source/locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -23,8 +23,8 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
-#: adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
+#: /home/katharina/adhocracy-plus/node_modules/adhocracy4/adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
+#: /home/katharina/adhocracy-plus/venv/lib/python3.8/site-packages/adhocracy4/categories/templates/a4categories/includes/module_categories_form.html:3
```

Comment:
does the 'gsed' check not work anymore?

Reply:
Oh, it should! Args. Why did I commit that? Thanks!

Reply:
@phillimorland

Reply:
it's easily done :p

Reply:
@phillimorland Yes, I think, I fucked up the rebase, but it's fixed now!